Dockerfile

```diff
@@ -1029,7 +1029,11 @@ ENV MW_AUTOUPDATE=true \
 	MW_DB_SERVER=db \
 	MW_DB_NAME=mediawiki \
 	MW_DB_USER=root \
+	MW_DB_PASS_FILE="/run/secrets/db_password /run/secrets/db_root_password" \
```

Comment:
here and for MW_DB_INSTALLDB_PASS_FILE the name of the secret suggests that the expectation is that the user is going to be `root` - I'd prefer to avoid assuming that. Can we have the installdb password at `db_install_password` instead?